starlark/int.go

```diff
@@ -26,7 +26,15 @@ func MakeInt64(x int64) Int {
 		}
 		return Int{&smallint[x]}
 	}
-	return Int{new(big.Int).SetInt64(x)}
+	if 0 <= x && int64(int32(x)) == x {
+		// x is guaranteed to fit into a single big.Word.
+		// Most starlark ints remain small,
+		// but math/big assumes that since you've chosen to use math/big,
```

Comment:
That seems like a mistake in big.Int. I bet even most multi-precision integers are zero, one, or some small positive value.

Reply:
Indeed. I'll do a bit of experimentation to see what a better approach from within math/big might be. There are quite a lot of options.

Reply:
For others reading this, this became https://golang.org/cl/150999. However, best case this lands in nine months.